docs/conf.py

```diff
@@ -105,18 +105,11 @@
 
 # The theme to use for HTML and HTML Help pages.  See the documentation for
 # a list of builtin themes.
-html_theme_path = ['_themes']
-html_theme = 'flask_small'
-
+html_theme = 'alabaster'
 
 # Theme options are theme-specific and customize the look and feel of a theme
 # further.  For a list of options available for each theme, see the
 # documentation.
-html_theme_options = {
```

Comment:
I removed the `html_theme_options` as the tests were failing (first because there was no `flask_small` theme, then because the `index_logo` option is not supported). I hope it's ok to do this (I took the alabaster theme from other invenio repos)?